gdnative/src/color.rs

```diff
@@ -0,0 +1,49 @@
+use sys;
+use get_api;
+use std::mem::transmute;
+
+#[derive(Clone, Copy)]
+pub struct Color {
+    pub r: f32,
+    pub g: f32,
+    pub b: f32,
+    pub a: f32,
+}
+
+impl Color {
+    pub fn rgba(r: f32, g: f32, b: f32, a: f32) -> Color {
+        Color { r, g, b, a }
+    }
+
+    pub fn rgb(r: f32, g: f32, b: f32) -> Color {
+        Color { r, g, b, a: 1.0 }
+    }
+
+    fn as_sys_color(&self) -> &sys::godot_color {
+        unsafe { transmute(self) }
```

Comment:
Needs repr(c) for Color?

Reply:
Woops! indeed, thanks. I'll add it tonight or tomorrow (unless you want to write the PR).

Reply:
Nah I'm happy with the backseat driving for now :)

Reply:
In any case, thanks for watching over this stuff and don't hesitate to point me to things you see that need fixing.